spec/dummy/config/application.rb

```diff
@@ -20,7 +20,7 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
+    # config.active_record.raise_in_transactional_callbacks = true
```

Comment:
[MISC] Why did you comment this?

Reply:
It's deprecated in rails 5.1 and removed in rails 5.2. c.f: https://github.com/rails/rails/issues/35671#issuecomment-474627832